planning/behavior_path_planner/src/utilities.cpp

```diff
@@ -1386,14 +1389,20 @@ PathPointWithLaneId insertStopPoint(double length, PathWithLaneId * path)
       break;
     }
   }
+  if (accumulated_length <= length) {
+    RCLCPP_ERROR_STREAM(
```

Comment:
@kosuke55 
can you change these to throttle 5000ms?

Reply:
Change canceled. When this error occurs, the node dies, so we must be sure to display the message.

Reply:
@kosuke55 
then you need throw exception for it.